js/stores/simulation-store.ts

```diff
@@ -453,6 +462,14 @@ export class SimulationStore {
     this.anyHotSpotDefinedByUser = val;
   }
 
+  @action.bound setKeyActive(val: boolean) {
```

Comment:
"Active" makes me think of the CSS ":active" state, i.e. selected. Perhaps something like `setKeyVisible`, `setKeyVisibility`, or `setKeyShowing`?

Reply:
True, I've changed that to `setKeyVisible`.